inc/3rd-party/plugins/autoptimize.php

```diff
@@ -103,9 +122,35 @@ function rocket_activate_autoptimize() {
 	if ( 'on' === get_option( 'autoptimize_css_defer' ) ) {
 		update_rocket_option( 'async_css', 0 );
 	}
+
+	$lazyload = get_option( 'autoptimize_imgopt_settings' );
+
+	if ( 1 === (int) $lazyload['autoptimize_imgopt_checkbox_field_3'] ) {
+		update_rocket_option( 'lazyload', 0 );
+		update_rocket_option( 'lazyload_iframes', 0 );
+		update_rocket_option( 'lazyload_youtube', 0 );
+	}
 }
 add_action( 'activate_autoptimize/autoptimize.php', 'rocket_activate_autoptimize', 11 );
 
+/**
+ * Disable WP Rocket lazyload fields if Autoptimize lazyload is enabled
+ *
+ * @since 3.3.4
+ * @author Remy Perona
+ *
+ * @return bool
+ */
+function rocket_maybe_disable_lazyload() {
```

Comment:
Shouldn't functions related to Autoptimize be named accordingly? Like `rocket_autoptimize_maybe_disable_lazyload()`.

Reply:
the previous functions were already named that way so I kept the same schema